command/server/config.go

```diff
@@ -68,6 +68,12 @@ type Config struct {
 
 	DisableSentinelTrace    bool        `hcl:"-"`
 	DisableSentinelTraceRaw interface{} `hcl:"disable_sentinel_trace"`
+
+	HostnameHeader    bool        `hcl:"-"`
```

Comment:
Could you add some minimal config tests to verify the new option, like augment `testLoadConfigFile`?

Reply:
Yep, will do